Documentation/packaging.md

```diff
@@ -56,7 +59,8 @@ To solve this, you can download the CoreOS PXE image before starting the build p
 
 ## Bundling with systemd
 
-Most Linux distributions don't allow to use pre-build binaries or to reuse copy of code that is already otherwise packaged. systemd falls in this category, as Debian and Fedora already package systemd and rkt needs systemd.
+Most Linux distributions don't allow to use prebuilt binaries or to reuse copy of code that is already otherwise packaged.
```

Comment:
Most Linux distributions don't allow the use of prebuilt binaries, or reuse of code that is already otherwise packaged.

(Meaning of this isn't entirely clear to me; I'll check context. In the meantime, correct as above to at least make it grammatical.)